core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/clone/CloneWorkspaceAwaitStorageContainerStep.scala

```diff
@@ -29,7 +29,7 @@ class CloneWorkspaceAwaitStorageContainerStep(
 )(implicit executionContext: ExecutionContext)
     extends WorkspaceCloningStep(workspaceRepository, monitorRecordDao, workspaceId, job) {
 
-  override val jobType: JobType = JobType.CloneWorkspaceContainerInit
+  override val jobType: JobType = JobType.CloneWorkspaceAwaitContainerResult
```

Comment:
This was a copy/paste error, but as far as I can tell, it had no effect because we never call `scheduleNextJob` from this step (since it is the last one).

Reply:
Oof, good catch.